conjure-java-core/src/test/java/com/palantir/conjure/java/UndertowServiceEteTest.java

```diff
@@ -587,12 +587,11 @@ public void testStringParametersDoNotChangeWhenUsingJsonAndJavaString() {
         } catch (RemoteException e) {
             // .getError() returns the SerializableError which should contain the legacy parameters sent over the wire.
             jsonParams = e.getError().parameters();
-            // e should be an instance of ErrorWithComplexArgsException, which has rich parameters as well.
-            //            assertThat(e.getCause()).isInstanceOfSatisfying(ErrorWithComplexArgsException.class, exception
-            // -> {
-            //                assertThat(exception.error().parameters().optionalExample().getOptionalString())
-            //                        .contains("optional-value");
-            //            });
+            // e should be an instance of ErrorWithComplexArgsException, which has rich parameters as well
+            assertThat(e.getCause()).isInstanceOfSatisfying(ErrorWithComplexArgsException.class, exception -> {
```

Comment:
Shouldn't this be
```suggestion
            assertThat(e).isInstanceOfSatisfying(ErrorWithComplexArgsException.class, exception -> {
```
rather than `e.getCause()`?

Reply:
Discussed about this yesterday. This is because of the pattern from https://github.com/palantir/dialogue/blob/c17a28c5673cd3988056a4181d38f5f1eae43f59/dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/DefaultClients.java#L148.

We ought to either replicate this pattern with the new remote exceptions, or instead get rid of it entirely (and do [what is done for other types of exceptions](https://github.com/palantir/dialogue/blob/c17a28c5673cd3988056a4181d38f5f1eae43f59/dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/DefaultClients.java#L134))